dotnet/src/SemanticKernel/CoreSkills/TextMemorySkill.cs

```diff
@@ -43,9 +43,22 @@ public class TextMemorySkill
     public const string LimitParam = "limit";
 
     private const string DefaultCollection = "generic";
-    private const string DefaultRelevance = "0.75";
+    private const string DefaultRelevance = "0.0";
     private const string DefaultLimit = "1";
 
+    /// <summary>
+    /// Creates a new instance of the TextMemorySkill
+    /// </summary>
+    /// <param name="collection">The default collection for Recall</param>
+    /// <param name="relevance">The default relevance value for Recall</param>
+    /// <param name="limit">The default limit for Recall</param>
+    public TextMemorySkill(string collection = DefaultCollection, string relevance = DefaultRelevance, string limit = DefaultLimit)
```

Comment:
users will need some help with the relevance param, e.g. what's the range? what's a good value? the function should probably validate it too

Reply:
Added comments from below to help here. There hadn't been validation elsewhere -- if they give a value greater than 1.0 it will fail to find memories. I'll lean on existing flexibility there for now.